server.js

```diff
@@ -0,0 +1,41 @@
+const http = require('http');
+const fs = require('fs');
+const PORT = process.env.PORT || 8080;
+
+
+const server = http.createServer((request, response) => {
+  let method = request.method;
+  let url = request.url;
+
+  if (url.indexOf('public') < 1) { //Add public to url
+    url = 'public' + request.url;
+  }
+
+  console.log(url);
+  console.log('Method: ', method);
+
```

Comment:
You will want your switch case to switch on a know set of variable, like method. We know we will only need to handle `GET`, `POST`, `PUT`, and `DELETE`, this is perfect for a switch. The `uri` can contain anything so we need the code to be more dynamic we responding to it.